connector/web3/web3.go

```diff
@@ -42,19 +56,20 @@ func (c *web3Connector) Verify(address, msg, signedMsg string) (identity connect
 		return identity, fmt.Errorf("could not decode hex string of signed nonce: %v", err)
 	}
 
+	msgHash := signHash([]byte(msg))
+
 	// This is the v parameter in the signature. Per the yellow paper, 27 means even and 28
 	// means odd.
 	if signb[64] == 27 || signb[64] == 28 {
 		signb[64] -= 27
```

Comment:
I would not modify the copy that we pass to `VerifyERC1271Signature`. That might have completely different logic that doesn't involve this byte shifting.

Let me see if we can get rid of this. I don't think we support Ledger anymore.